src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
+                    fileTypes: ['pdf', 'jpg', 'jpeg', 'tiff', 'tif', 'png'],
+                    maxSize: 27262976, // TODO: create constant
+                    createPayload: (file) => {
+                      const payload = new FormData();
+                      payload.append('preneed_attachment[file_data]', file); // TODO: update this with correct property (e.g. 'health_record[file_data]')
+
+                      return payload;
+                    },
+                    parseResponse: (response, file) => {
+                      return {
+                        name: file.name,
+                        confirmationCode: response.data.attributes.guid
+                      };
+                    },
+                    attachmentSchema: {
+                      'ui:title': 'Document type'
+                    },
+                    attachmentName: {
+                      'ui:title': 'Document name'
+                    }
+                  }),
+                  { 'ui:description': () => (<div>
+                    <p>File upload guidelines:</p>
+                    <ul>
+                      <li>File types you can upload: .pdf, .jpeg, .gif, .tiff, or .png</li>
+                      <li>Maximum file size: 25 MB</li>
+                    </ul>
+                    <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+                  </div>) })
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    medicalRecords: {
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        properties: {
+                          name: {
+                            type: 'string'
+                          },
+                          size: {
+                            type: 'integer'
+                          },
+                          confirmationCode: {
+                            type: 'string'
+                          },
+                          attachmentId: {
+                            type: 'string',
+                            'enum': [
+                              '1',
+                              '2',
+                              '3',
+                              // '4',
+                              '5',
+                              '6'
+                            ],
+                            enumNames: [
+                              'Discharge',
+                              'Marriage related',
+                              'Dependent related',
+                              // 'VA preneed form',
+                              'Letter',
+                              'Other'
```

Comment:
Are these all accurate and appropriate to this form?

Reply:
These are filler until the correct types are determined, as @goldenmeanie has indicated these are still TBD.